### ISSUE TITLE
Workspace Grid causes stacktraces

### ISSUE THREAD

**Issue by derjanb** [zakkak/workspace-grid#76]
Gnome crashes right after the extension is enabled and the current session is closed. Repeated logins show the desktop for 1-3 seconds and I'm logged out again.
It makes no difference whether other shell extensions are running or not.
This happens to me if the session is started via LightDM. It's working fine if started via `startx` from command line.

metadata.json:
``` json
{
  "_generated": "Generated by SweetTooth, do not edit", 
  "description": "Arranges workspaces in a configurable grid.\nAlso:\n* implements keybindings for left/right workspace navigation (up/down are already implemented)\n* updates workspaces sidebar with grid configuration (use Remove Workspaces Sidebar if you don't want it).\n\nYou may also wish to consider instead of/with this extension: Frippery Bottom Panel (already has workspace grid functionality if you want a bottom panel); Frippery Static Workspaces (holds the number of workspaces static and skips all the workspace grid stuff); Remove Workspaces Sidebar; Workspace Indicator Extension (textual indicator/workspace switcher); WorkspaceBar (graphical indicator/workspace switcher); Workspace navigator extension (use arrow keys to navigate workspaces in the Overview).\nSee homepage for more details.", 
  "dev-version": "1.5.0", 
  "extension-id": "workspace-grid", 
  "gettext-domain": "workspace-grid", 
  "name": "Workspace Grid", 
  "settings-schema": "org.gnome.shell.extensions.workspace-grid", 
  "shell-version": [
    "3.28"
  ], 
  "url": "https://github.com/zakkak/workspace-grid-gnome-shell-extension", 
  "uuid": "workspace-grid@mathematical.coffee.gmail.com", 
  "version": 27
}
```

```log
Jul 13 15:11:15 norm gnome-shell[3460]: Some code accessed the property 'MAX_THUMBNAIL_SCALE' on the module 'workspaceThumbnail'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Jul 13 15:11:15 norm gnome-shell[3460]: Some code accessed the property 'ThumbnailState' on the module 'workspaceThumbnail'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Jul 13 15:11:15 norm gnome-shell[3460]: Object St.Bin (0x5576b9d71240), has been already deallocated - impossible to access to it. This might be caused by the fact that the object has been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
Jul 13 15:11:15 norm gnome-shell[3460]: Object St.Bin (0x5576b9d6de80), has been already deallocated - impossible to access to it. This might be caused by the fact that the object has been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: == Stack trace for context 0x5576b869c340 ==
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #0 0x5576b8a8e9c0 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:947 (0x7f675d2aa2b8 @ 275)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #1 0x5576b8a8e928 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:937 (0x7f675d2aa230 @ 128)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #2 0x5576b8a8e880 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:1026 (0x7f675d2aa3c8 @ 164)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #3 0x5576b8a8e7e8 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:1264 (0x7f675d2aaa28 @ 90)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #4 0x5576b8a8e738 i   resource:///org/gnome/shell/ui/extensionSystem.js:129 (0x7f67c415a1a8 @ 408)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #5 0x5576b8a8e6a0 i   resource:///org/gnome/shell/ui/extensionSystem.js:171 (0x7f67c415a2b8 @ 241)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #6 0x5576b8a8e620 i   resource:///org/gnome/shell/ui/extensionSystem.js:318 (0x7f67c415a9a0 @ 13)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #7 0x7ffe69e41e80 b   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f67c44d3230 @ 386)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #8 0x5576b8a8e540 i   resource:///org/gnome/shell/misc/extensionUtils.js:191 (0x7f67c415e120 @ 459)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #9 0x7ffe69e42af0 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f67c44b5de0 @ 71)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #10 0x5576b8a8e4b0 i   resource:///org/gnome/shell/misc/extensionUtils.js:197 (0x7f67c415e230 @ 28)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #11 0x7ffe69e437f0 b   resource:///org/gnome/shell/misc/fileUtils.js:27 (0x7f67c415e340 @ 323)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #12 0x5576b8a8e420 i   resource:///org/gnome/shell/misc/extensionUtils.js:196 (0x7f67c415e1a8 @ 72)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #13 0x7ffe69e44450 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f67c44b5de0 @ 71)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #14 0x5576b8a8e390 i   resource:///org/gnome/shell/ui/extensionSystem.js:320 (0x7f67c415a918 @ 201)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #15 0x5576b8a8e310 i   resource:///org/gnome/shell/ui/extensionSystem.js:328 (0x7f67c415aa28 @ 36)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #16 0x5576b8a8e290 i   resource:///org/gnome/shell/ui/extensionSystem.js:359 (0x7f67c415ac48 @ 67)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #17 0x5576b8a8e208 i   resource:///org/gnome/shell/ui/extensionSystem.js:367 (0x7f67c415acd0 @ 41)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #18 0x5576b8a8e178 i   resource:///org/gnome/shell/ui/main.js:229 (0x7f67c44da340 @ 1152)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #19 0x5576b8a8e0f0 i   resource:///org/gnome/shell/ui/main.js:133 (0x7f67c44da230 @ 237)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #20 0x5576b8a8e078 i   <main>:1 (0x7f67c4486b38 @ 48)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: == Stack trace for context 0x5576b869c340 ==
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #0 0x5576b8a8e9c0 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:948 (0x7f675d2aa2b8 @ 304)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #1 0x5576b8a8e928 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:937 (0x7f675d2aa230 @ 128)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #2 0x5576b8a8e880 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:1026 (0x7f675d2aa3c8 @ 164)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #3 0x5576b8a8e7e8 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:1264 (0x7f675d2aaa28 @ 90)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #4 0x5576b8a8e738 i   resource:///org/gnome/shell/ui/extensionSystem.js:129 (0x7f67c415a1a8 @ 408)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #5 0x5576b8a8e6a0 i   resource:///org/gnome/shell/ui/extensionSystem.js:171 (0x7f67c415a2b8 @ 241)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #6 0x5576b8a8e620 i   resource:///org/gnome/shell/ui/extensionSystem.js:318 (0x7f67c415a9a0 @ 13)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #7 0x7ffe69e41e80 b   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f67c44d3230 @ 386)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #8 0x5576b8a8e540 i   resource:///org/gnome/shell/misc/extensionUtils.js:191 (0x7f67c415e120 @ 459)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #9 0x7ffe69e42af0 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f67c44b5de0 @ 71)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #10 0x5576b8a8e4b0 i   resource:///org/gnome/shell/misc/extensionUtils.js:197 (0x7f67c415e230 @ 28)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #11 0x7ffe69e437f0 b   resource:///org/gnome/shell/misc/fileUtils.js:27 (0x7f67c415e340 @ 323)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #12 0x5576b8a8e420 i   resource:///org/gnome/shell/misc/extensionUtils.js:196 (0x7f67c415e1a8 @ 72)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #13 0x7ffe69e44450 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f67c44b5de0 @ 71)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #14 0x5576b8a8e390 i   resource:///org/gnome/shell/ui/extensionSystem.js:320 (0x7f67c415a918 @ 201)
Jul 13 15:11:15 norm gnome-shell[3460]: Object St.Bin (0x5576b9d6de80), has been already finalized. Impossible to set any property to it.
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #15 0x5576b8a8e310 i   resource:///org/gnome/shell/ui/extensionSystem.js:328 (0x7f67c415aa28 @ 36)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #16 0x5576b8a8e290 i   resource:///org/gnome/shell/ui/extensionSystem.js:359 (0x7f67c415ac48 @ 67)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #17 0x5576b8a8e208 i   resource:///org/gnome/shell/ui/extensionSystem.js:367 (0x7f67c415acd0 @ 41)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #18 0x5576b8a8e178 i   resource:///org/gnome/shell/ui/main.js:229 (0x7f67c44da340 @ 1152)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #19 0x5576b8a8e0f0 i   resource:///org/gnome/shell/ui/main.js:133 (0x7f67c44da230 @ 237)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #20 0x5576b8a8e078 i   <main>:1 (0x7f67c4486b38 @ 48)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: == Stack trace for context 0x5576b869c340 ==
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #0 0x5576b8a8e880 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:1032 (0x7f675d2aa3c8 @ 193)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #1 0x5576b8a8e7e8 i   /home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:1264 (0x7f675d2aaa28 @ 90)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #2 0x5576b8a8e738 i   resource:///org/gnome/shell/ui/extensionSystem.js:129 (0x7f67c415a1a8 @ 408)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #3 0x5576b8a8e6a0 i   resource:///org/gnome/shell/ui/extensionSystem.js:171 (0x7f67c415a2b8 @ 241)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #4 0x5576b8a8e620 i   resource:///org/gnome/shell/ui/extensionSystem.js:318 (0x7f67c415a9a0 @ 13)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #5 0x7ffe69e41e80 b   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f67c44d3230 @ 386)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #6 0x5576b8a8e540 i   resource:///org/gnome/shell/misc/extensionUtils.js:191 (0x7f67c415e120 @ 459)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #7 0x7ffe69e42af0 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f67c44b5de0 @ 71)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #8 0x5576b8a8e4b0 i   resource:///org/gnome/shell/misc/extensionUtils.js:197 (0x7f67c415e230 @ 28)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #9 0x7ffe69e437f0 b   resource:///org/gnome/shell/misc/fileUtils.js:27 (0x7f67c415e340 @ 323)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #10 0x5576b8a8e420 i   resource:///org/gnome/shell/misc/extensionUtils.js:196 (0x7f67c415e1a8 @ 72)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #11 0x7ffe69e44450 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f67c44b5de0 @ 71)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #12 0x5576b8a8e390 i   resource:///org/gnome/shell/ui/extensionSystem.js:320 (0x7f67c415a918 @ 201)
Jul 13 15:11:15 norm gsd-wacom[3538]: Theme parsing error: gtk.css:5:20: 'foreground-color' is not a valid property name
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #13 0x5576b8a8e310 i   resource:///org/gnome/shell/ui/extensionSystem.js:328 (0x7f67c415aa28 @ 36)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #14 0x5576b8a8e290 i   resource:///org/gnome/shell/ui/extensionSystem.js:359 (0x7f67c415ac48 @ 67)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #15 0x5576b8a8e208 i   resource:///org/gnome/shell/ui/extensionSystem.js:367 (0x7f67c415acd0 @ 41)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #16 0x5576b8a8e178 i   resource:///org/gnome/shell/ui/main.js:229 (0x7f67c44da340 @ 1152)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #17 0x5576b8a8e0f0 i   resource:///org/gnome/shell/ui/main.js:133 (0x7f67c44da230 @ 237)
Jul 13 15:11:15 norm org.gnome.Shell.desktop[3460]: #18 0x5576b8a8e078 i   <main>:1 (0x7f67c4486b38 @ 48)```

**Comment by pachulo** [comment]
I'm running Ubuntu 18.04.1 with GDM and Wayland and I'm observing the same symptoms.

**Comment by konrad-garus** [comment]
Ubuntu here, same issue. It's gigabytes of log files (journal) on my precious tiny SSD.

**Comment by flamders** [comment]
Same error. Has noted it for a few months. One obvious error is that this._porthole is null (workspace-grid@mathematical.coffee.gmail.com/extension.js). I.e., this._porthole variable was never set:

JS ERROR: TypeError: this._porthole is null#012ThumbnailsBox<._getPreferredHeight@/home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:680:1#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_init/<@resource:///org/gnome/shell/ui/layout.js:209:17#012getPageY@resource:///org/gnome/shell/ui/iconGrid.js:916:24#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_diffToPage@resource:///org/gnome/shell/ui/appDisplay.js:635:49#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012goToPage@resource:///org/gnome/shell/ui/appDisplay.js:608:26#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_init/<@resource:///org/gnome/shell/ui/appDisplay.js:435:49#012_emit@resource:///org/gnome/gjs/modules/signals.js:128:27#012_hideDone@resource:///org/gnome/shell/ui/overview.js:647:9#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_callOnFunction@resource:///org/gnome/gjs/modules/tweener/tweener.js:208:13#012_updateTweenByIndex@resource:///org/gnome/gjs/modules/tweener/tweener.js:337:9#012_updateTweens@resource:///org/gnome/gjs/modules/tweener/tweener.js:350:18#012_onEnterFrame@resource:///org/gnome/gjs/modules/tweener/tweener.js:365:10#012_emit@resource:///org/gnome/gjs/modules/signals.js:128:27#012_onNewFrame@resource:///org/gnome/shell/ui/tweener.js:244:9#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_init/<@resource:///org/gnome/shell/ui/tweener.js:219:13 
null#012ThumbnailsBox<._getPreferredHeight@/home/user/.local/share/gnome-shell/extensions/workspace-grid@mathematical.coffee.gmail.com/extension.js:680:1#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_init/<@resource:///org/gnome/shell/ui/layout.js:209:17#012getPageY@resource:///org/gnome/shell/ui/iconGrid.js:916:24#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_diffToPage@resource:///org/gnome/shell/ui/appDisplay.js:635:49#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012goToPage@resource:///org/gnome/shell/ui/appDisplay.js:608:26#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_init/<@resource:///org/gnome/shell/ui/appDisplay.js:435:49#012_emit@resource:///org/gnome/gjs/modules/signals.js:128:27#012_hideDone@resource:///org/gnome/shell/ui/overview.js:647:9#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_callOnFunction@resource:///org/gnome/gjs/modules/tweener/tweener.js:208:13#012_updateTweenByIndex@resource:///org/gnome/gjs/modules/tweener/tweener.js:337:9#012_updateTweens@resource:///org/gnome/gjs/modules/tweener/tweener.js:350:18#012_onEnterFrame@resource:///org/gnome/gjs/modules/tweener/tweener.js:365:10#012_emit@resource:///org/gnome/gjs/modules/signals.js:128:27#012_onNewFrame@resource:///org/gnome/shell/ui/tweener.js:244:9#012wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22#012_init/<@resource:///org/gnome/shell/ui/tweener.js:219:13 

Which is probably related to the second log trace with workspaceThumbnail.js:

Sep 21 15:42:45 computer gnome-shell[2154]: clutter_box_layout_set_spacing: assertion 'CLUTTER_IS_BOX_LAYOUT (layout)' failed 
Sep 21 15:42:45 computer gnome-shell[2154]: message repeated 7 times: [ clutter_box_layout_set_spacing: assertion 'CLUTTER_IS_BOX_LAYOUT (layout)' failed] 
Sep 21 15:42:45 computer gnome-shell[2154]: Object St.Bin (0x55b4cd799940), has been already deallocated - impossible to access to it. This might be caused by the fact that the object has been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: == Stack trace for context 0x55b4cc173330 == 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #0 0x7ffdcdbcc960 b   resource:///org/gnome/shell/ui/workspaceThumbnail.js:974 (0x7f4d841f5560 @ 439) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #1 0x7ffdcdbcdd60 b   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f4d846b5de0 @ 71) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #2 0x55b4cc51d498 i   resource:///org/gnome/shell/ui/workspaceThumbnail.js:884 (0x7f4d841f52b8 @ 390) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #3 0x7ffdcdbcf0d0 b   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f4d846b5de0 @ 71) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #4 0x7ffdcdbcf190 b   self-hosted:916 (0x7f4d846f12b8 @ 367) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #5 0x7ffdcdbcf280 b   resource:///org/gnome/gjs/modules/signals.js:128 (0x7f4d846d2230 @ 386) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #6 0x55b4cc51d400 i   resource:///org/gnome/shell/ui/overview.js:568 (0x7f4d843cf890 @ 324) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #7 0x7ffdcdbcfed0 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f4d846b5de0 @ 71) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #8 0x55b4cc51d380 i   resource:///org/gnome/shell/ui/overview.js:540 (0x7f4d843cf808 @ 109) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #9 0x7ffdcdbd0ab0 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f4d846b5de0 @ 71) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #10 0x55b4cc51d300 i   resource:///org/gnome/shell/ui/overview.js:670 (0x7f4d843cfb38 @ 71) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #11 0x7ffdcdbd1690 I   resource:///org/gnome/gjs/modules/_legacy.js:82 (0x7f4d846b5de0 @ 71) 
Sep 21 15:42:45 computer org.gnome.Shell.desktop[2154]: #12 0x55b4cc51d280 i   resource:///org/gnome/shell/ui/main.js:197 (0x7f4d846da3c8 @ 39) 
Sep 21 15:42:45 computer gnome-shell[2154]: Object St.Bin (0x55b4cd799940), has been already deallocated - impossible to access to it. This might be caused by the fact that the object has been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs

**Comment by zakkak** [comment]
The porthole issue has been resolved in e28f180fd645d272cffabbc25a1ddee4277efd7a

The other issue with the ST.Bin objects seems to gets fixed with this patch https://github.com/zakkak/workspace-grid/issues/86#issuecomment-424869349 , waiting for a PR from the author to get it upstream.

**Comment by zakkak** [comment]
Fixed in branch 3.28 by #92 and https://github.com/zakkak/workspace-grid/commit/e28f180fd645d272cffabbc25a1ddee4277efd7a
